extensions/amp-animation/0.1/web-animation-types.js

```diff
@@ -168,6 +168,8 @@ const WHITELISTED_RPOPS = {
   'opacity': true,
   'transform': true,
   'visibility': true,
+  'offset-distance': true,
```

Comment:
Both forms are currently allowed and we leave it up to the animations API to convert. We could convert ourselves, in which case we'd only need one.